include/deal.II/lac/petsc_precondition.h

```diff
@@ -146,6 +146,14 @@ namespace PETScWrappers
                         const AdditionalData &additional_data = AdditionalData());
 
     /**
+     * Same as above but without setting a matrix to form the preconditioner.
+     * Intended to be used with SLEPc objects.
+     */
+    PreconditionJacobi (const MPI_Comm communicator,
+                        const AdditionalData &additional_data = AdditionalData());
```

Comment:
i can catch this case nicely https://github.com/davydden/dealii/commit/babe68f61d1fb4bd4c279ae547576516fa165ddd